dot/rpc/modules/chain.go

```diff
@@ -45,11 +46,16 @@ type ChainBlock struct {
 
 // ChainBlockHeaderResponse struct
 type ChainBlockHeaderResponse struct {
-	ParentHash     string   `json:"parentHash"`
-	Number         *big.Int `json:"number"`
-	StateRoot      string   `json:"stateRoot"`
-	ExtrinsicsRoot string   `json:"extrinsicsRoot"`
-	Digest         [][]byte `json:"digest"`
+	ParentHash     string                 `json:"parentHash"`
+	Number         *big.Int               `json:"number"`
+	StateRoot      string                 `json:"stateRoot"`
+	ExtrinsicsRoot string                 `json:"extrinsicsRoot"`
+	Digest         ChainBlockHeaderDigest `json:"digest"`
```

Comment:
is this in the spec? I couldn't find it anywhere

Reply:
I'm using this as spec for JSON result: https://github.com/w3f/PSPs/blob/psp-rpc-api/psp-002.md#chain_getHeader 

But I see now that I should be treating number as a hex string, so I'll update that.